app/models/answer.rb

```diff
@@ -30,7 +30,6 @@ class Answer < ApplicationRecord
     :storage => :s3,
     :s3_permissions => :private
 
-  validates :text, presence: true
```

Comment:
why are we getting rid of this validation

Reply:
we are defaulting answers to empty